### ISSUE TITLE
hdf5 detection in configure

### ISSUE THREAD

**Issue by ipelupessy** [amusecode/amuse#347]
at the moment only checks standard locations, it could use pkg-config ?

**Comment by jhidding** [comment]
If `pkg-config` fails, the following should always work:

```make
hdf5_cflags = $(shell h5c++ -show '%' | cut -d% -f1 | cut -d' ' -f2-) -I$(hdf5_root)/include
hdf5_libs = $(shell h5c++ -show '%' | cut -d% -f2 | cut -d' ' -f2-)
```

**Comment by rieder** [comment]
This isn't fixed yet - makes simplex build fail on macOS...

**Comment by rieder** [comment]
Fixed by #453
